content/en/01-about-pixie/04-architecture.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Architecture"
+metaTitle: "About Pixie | Architecture"
+metaDescription: "What Pixie deploys to your cluster and what each component does."
+order: 4 
+---
+
+## Main Components 
+
+Pixie's deploy consists of three main components:
+1. Vizier
```

Comment:
We may want to use the same language for some of these as we do on the About Pixie page, so that they don't see different info in different places. 

I think a user might be confused about PEM vs Vizier